appeals/api/src/server/utils/horizon-gateway.js

```diff
@@ -107,19 +107,29 @@ export const getAppealFromHorizon = async (caseReference) => {
 	const appealData = await got
 		.post(url, {
 			json: requestBody,
-			parseJson: (data) => parseHorizonGetCaseResponse(data)
+			parseJson: (data) => parseHorizonGetCaseResponse(data),
+			timeout: {
+				request: 5000
+			}
 		})
 		.json()
 		.catch((error) => {
-			if (error.response.body && JSON.stringify(error.response.body).includes('faultstring')) {
+			if (error.code === 'ETIMEDOUT') {
+				logger.error('Call to Horizon timed-out');
+			} else if (
+				error.response.body &&
+				JSON.stringify(error.response.body).includes('faultstring')
```

Comment:
I would probably stringify this outside the if/then and reuse

Reply:
Agreed